app/components/media-popover.js

```diff
@@ -6,6 +6,18 @@ import HoverIntentMixin from 'client/mixins/hover-intent';
 export default Component.extend(HoverIntentMixin, {
   hoverTimeout: 0,
 
+  mediaYear: computed('media.startDate', function() {
+    const startDate = get(this, 'media.startDate');
+    if (startDate) {
+      return get(this, 'media.startDate').year();
```

Comment:
We should probably return something here to be safe.

`return startDate ? get(this, 'media.startDate').year() : ''`